README.md

```diff
@@ -49,6 +49,17 @@ resource "lxd_container" "test1" {
 }
 ```
 
+You can also launch a container directly from a remote image, not locally cached, by referencing the remote image name using the format:
```

Comment:
bah. I had too many images and got them mixed up :)

I flushed everything and retried. Aliases aren't set by this feature, so no harm done at all. Sorry about that!